### ISSUE TITLE
Is bomi under development?

### ISSUE THREAD

**Issue by perpi** [xylosper/bomi#452]
Is it still a cutting-edge player?
As I see in repo, it has not been touched for a couple of months!

**Comment by ahjolinna** [comment]
the dev took a [sabbatical](https://twitter.com/bomi_player/status/618745103587282945) from the project..I don't know for how long, I and @M4rtinK is searching at least a tmp dev for bomi...for now without a any luck

**Comment by M4rtinK** [comment]
Yep, I've already tried asking a few people with Qt/KDE background but so
far without luck. It is unfortunately not an easy task (bundled MPV,
etc.)...

2016-01-21 16:16 GMT+01:00 Alexander Waldemar Ahjolinna <
notifications@github.com>:

**Comment by varlesh** [comment]
Any news? :disappointed:
Maybe anybody fork project and merge pulls. Also @journalehsan commited for last mpv version 268b3b58fe5672f76860c3c408116978db038fb5
Where Qt-developers???

**Comment by varlesh** [comment]
OMG!
https://twitter.com/bomi_player/status/697646986560872450
